src/JsonBodyListener.php

```diff
@@ -51,4 +57,12 @@ public function onKernelRequest(GetResponseEvent $event)
 
         $request->request = new ParameterBag($data);
     }
+
+    private function requestFormatViolateSupportedFormats($format, $supportedFormats)
+    {
+        return null !== $format
+            && false !== $supportedFormats
```

Comment:
do you mean `false !== is_array($supportedFormats)`?

Reply:
nop, false is the default value I set when keys is missing

Reply:
indeed 👍